packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -28,13 +31,26 @@ export function RenderConfig({
     // We likely want to render something nicer eventually.
     throw result.error
   }
-  const globalContext: GlobalContext = { config: result.data, COMPONENTS }
+  const globalContext: GlobalContext = {
+    config: result.data,
+    COMPONENTS,
+    FrameWrapper,
+  }
 
   return (
     <>
-      {config.frames.map((frame, i) => (
-        <Frame key={i} frame={frame} globalContext={globalContext} />
-      ))}
+      {config.frames.map((frame, i) => {
+        const configPath = `workflow.frames.${i}`
```

Comment:
Having the the path to this frame in the config I think could be helpful for debugging. For example, we could tell you where in the config a component that is not configured correctly is.